README.markdown

```diff
@@ -42,8 +42,8 @@ you can change the target platform for your build straight from your command lin
 sbt compile -Dsbt.javacpp.platform="android-arm android-x86"
 ```
 
-In case you want to select a different javacpp version:
+In case you want to select a different javacpp version, add the following line in your `project/plugins.sbt`:
 
 ```scala
-javaCppVersion := "1.4.3"
+libraryDependencies += "org.bytedeco" % "javacpp" % "1.5.1"
```

Comment:
I'd much prefer the mechanism for specifying a different javacpp version to remain the same as it is now: setting a new version, rather than straight up requiring someone to declare the whole dependency. It's both simpler and backwards compatible.